src/components/AppBar/Logo.js

```diff
@@ -30,8 +32,17 @@ const Tagline = styled.p`
 `;
 
 function Logo() {
+  const { animateAndRedirect } = useContext(LandingPageAnimationContext);
+
+  function handleClick(e) {
+    e.preventDefault();
+    if (window.location.pathname !== ROOT_PATH) {
+      animateAndRedirect(ROOT_PATH);
+    }
+  }
+
   return (
-    <Link to="/" data-testid="logo">
+    <Link onClick={handleClick} to={ROOT_PATH} data-testid="logo">
```

Comment:
I left it as it is for accessibility purposes (user can see redirect URL) and possible better SEO. I think logos are usually wrapped in `<a>` tag rather than `<button>`.

Reply:
Makes sense. Just was wondering if providing _onClick_ handler and _to_ prop won't trigger redirect and therefore rendering twice.

Reply:
It won't redirect twice because of existence `e.preventDefault()`. 
This component is rendered as regular `<a>` with `href` tag, but redirection is delayed due to animation.

```
<a data-testid="logo" class="sc-bwzfXH ipdLkX" href="/">
```



![image](https://user-images.githubusercontent.com/14184100/61217824-d9fc0b80-a710-11e9-853b-52e3d341ca96.png)